src/Components/Quizz/StarsQuizz.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './StarsQuizz.css';
+
+const StarsQuizz = ({difficulty}) => {
+
+    const difHard = <ul><li className='etoilePleine'></li><li className='etoilePleine'></li><li className='etoilePleine'></li>
+    <li className='etoilePleine'></li><li className='etoilePleine'></li></ul>;
+    const difMedium = <ul><li className='etoilePleine'></li><li className='etoilePleine'></li><li className='etoileMi'></li>
+    <li className='etoileVide'></li><li className='etoileVide'></li></ul>
+    const difEasy = <ul><li className='etoileVide'></li><li className='etoileVide'></li><li className='etoileVide'></li>
+    <li className='etoileVide'></li><li className='etoileVide'></li></ul>
+
+    
+
+    return (
+        <div className='divEtoile'>
+        { difficulty === 'easy' ? difEasy : <p></p> }
+        { difficulty === 'medium' ? difMedium : <p></p> }
+        { difficulty === 'hard' ? difHard : <p></p> }
```

Comment:
Instead of displaying empty `<p></p>`, display `null` or use `&&` operator instead of the ternary operator `a ? b : c`. Or apply the above suggestion.

Also, fix indentation.